### PR TITLE
docs: update docker install for macos

### PR DESCRIPTION
**Pull request by gh-determined-ai**
## Description

Update Docker installation instructions for MacOS.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: docs/sysadmin-deploy-on-prem/requirements.txt [hunk 1]
```diff
@@ -15,51 +15,50 @@ A Determined cluster has the following requirements.
 Software
 ========
 
--  The Determined agent and master nodes must be configured with Ubuntu 16.04, Ubuntu 18.04, or
-   CentOS 7, or with macOS 10.13 or higher.
+-  The Determined agent and master nodes must be configured with Ubuntu 16.04 or higher, CentOS 7,
```
Comment (vishnu2kmohan) [inline docs/sysadmin-deploy-on-prem/requirements.txt:18]:
```suggestion
-  The Determined agent and master nodes must be configured with Ubuntu 20.04 or higher, CentOS 7,
```

File: docs/sysadmin-deploy-on-prem/requirements.txt [hunk 2]
```diff
@@ -15,51 +15,50 @@ A Determined cluster has the following requirements.
...
+   or macOS 10.13 or higher.
 
 -  The agent nodes must have :ref:`Docker installed <install-docker>`.
 
 -  To run jobs with GPUs, the Nvidia drivers must be installed on each Determined agent. Determined
-   requires version >= 384.81 of the Nvidia drivers. (The Nvidia drivers can be installed as part of
-   installing CUDA, but the rest of the CUDA toolkit is not required.)
+   requires a version greater than or equal to 384.81 of the Nvidia drivers. The Nvidia drivers can
```
Comment (vishnu2kmohan) [inline docs/sysadmin-deploy-on-prem/requirements.txt:24]:
```suggestion
   requires a version greater than or equal to 450.80 of the Nvidia drivers. The Nvidia drivers can
```

File: docs/sysadmin-deploy-on-prem/requirements.txt [hunk 3]
```diff
@@ -15,51 +15,50 @@ A Determined cluster has the following requirements.
...
-Docker is a dependency of several components of the Determined system. For example, every agent node
-must have Docker installed to allow it to run containerized workloads.
+Docker is a dependency of several Determined system components. For example, every agent node must
+have Docker installed to run containerized workloads.
 
 Install on Linux
 ================
 
-#. Install Docker. We require the version of Docker 19.03 or newer on the machine where the agent is
+#. Install Docker. Docker version 19.03 or newer is required on the machine where the agent is
```
Comment (vishnu2kmohan) [inline docs/sysadmin-deploy-on-prem/requirements.txt:56]:
```suggestion
#. Install Docker. Docker version 20.10 or newer is required on the machine where the agent is
```

**Review by vishnu2kmohan — APPROVED** [review]
Made a few suggestions to bump the minimum versions for Ubuntu, NVIDIA Drivers and Docker Engine.

File: docs/sysadmin-deploy-on-prem/requirements.txt [hunk 4]
```diff
@@ -15,51 +15,50 @@ A Determined cluster has the following requirements.
...
 
 -  The agent nodes must have :ref:`Docker installed <install-docker>`.
 
 -  To run jobs with GPUs, the Nvidia drivers must be installed on each Determined agent. Determined
-   requires version >= 384.81 of the Nvidia drivers. (The Nvidia drivers can be installed as part of
-   installing CUDA, but the rest of the CUDA toolkit is not required.)
+   requires a version greater than or equal to 450.80 of the Nvidia drivers. The Nvidia drivers can
+   be installed as part of a CUDA installation but the rest of the CUDA toolkit is not required.
+
+-  Determined supports the `supported Python versions <https://endoflife.date/python>`__.
```
Comment (ioga) [inline docs/sysadmin-deploy-on-prem/requirements.txt:27]: "supports the supported" seems like a bad wording to me, looks good otherwise.

**Review by ioga — APPROVED** [review]

**Pull request merged**